docs/codegen-options.md

```diff
@@ -0,0 +1,77 @@
+| `generateToString`                              | Boolean                                            | False                                         | Specifies whether generated model classes should have toString method defined. |
+| `modelNamePrefix`                               | String                                             | Empty                                         | Sets the prefix for GraphQL model classes (type, input, interface, enum, union). |
+| `modelNameSuffix`                               | String                                             | Empty                                         | Sets the suffix for GraphQL model classes (type, input, interface, enum, union). |
+| `modelValidationAnnotation`                     | String                                             | @javax.validation.<br>constraints.NotNull     | Annotation for mandatory (NonNull) fields. Can be null/empty. |
+| `customTypesMapping`                            | Map(String,String)                                 | Empty                                         | Can be used to supply custom mappings for scalars. <br/> Supports:<br/> * Map of (GraphqlObjectName.fieldName) to (JavaType) <br/> * Map of (GraphqlType) to (JavaType) |
+| `customAnnotationsMapping`                      | Map(String,String)                                 | Empty                                         | Can be used to supply custom annotations (serializers) for scalars. <br/> Supports:<br/> * Map of (GraphqlObjectName.fieldName) to (JavaAnnotation) <br/> * Map of (GraphqlType) to (JavaAnnotation) |
+| `fieldsWithResolvers`                           | Set(String)                                        | Empty                                         | Fields that require Resolvers should be defined here in format: `TypeName.fieldName` or `TypeName`. |
+| `fieldsWithoutResolvers`                        | Set(String)                                        | Empty                                         | Fields that DO NOT require Resolvers should be defined here in format: `TypeName.fieldName` or `TypeName`. Can be used in conjunction with `generateExtensionFieldsResolvers` option. |
+| `generateParameterizedFieldsResolvers`          | Boolean                                            | True                                          | If true, then generate separate `Resolver` interface for parametrized fields. If false, then add field to the type definition and ignore field parameters. |
+| `generateExtensionFieldsResolvers`              | Boolean                                            | False                                         | Specifies whether all fields in extensions (<code>extend type</code> and <code>extend interface</code>) should be present in Resolver interface instead of the type class itself. |
```

Comment:
Maybe we can simplify `<code>` using backticks instead. That's totally unnecessary nitpicking (I'm sure I have undiagnosed OCD) 😄

Reply:
Thanks!